docs/src/components/range-input.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Range input
+mainTitle: Components
+layout: main.pug
+category: Components
+withHeadings: true
+navWeight: 13
+editable: true
+githubSource: docs/src/components/range-input.md
+---
```

Comment:
should we have a flag for non-released components @rayrutjes, same for `ais-menu`, or do we just plan to release soon? 

see #346 too

Reply:
We should only release docs when releasing the library, this is my bad. I would personally not develop anything specific.